README.md

```diff
@@ -354,6 +315,49 @@ checkpointing so the job restarts near where it was interrupted.
     python3 xpk.py workload list \
     --cluster xpk-test --filter-by-job=$USER
     ```
+## Pathways on XPK
+
+* Cluster Create for Pathways:
+    Pathways compatible cluster can be created using `--enable-pathways`
+    ```shell
+    python3 xpk.py cluster create \
+    --cluster xpk-pw-test \
+    --num-slices=4 --on-demand \
+    --tpu-type=v5litepod-16 \
+    --enable-pathways
+    ```
+
+*   Workload Create for Pathways:
+    Pathways workload can be submitted using `--use-pathways` on a Pathways enabled cluster (created with `--enable-pathways`)
+    
+    Pathways workload example:
+    ```shell
+    python3 xpk.py workload create \
+    --workload xpk-pw-test \
+    --num-slices=1 \
+    --tpu-type=v5litepod-16 \
+    --cluster xpk-pw-test \
+    --use-pathways \
+    --server-image=<Pathways server image> \
+    --proxy-server-image=<Pathways proxy server image> \
+    --docker-name='user-workload' \
+    --docker-image=<maxtext docker image> \
+    --command='bash /usr/pathways/ifrt/maxtext_entrypoint.sh base_output_directory=<output directory> dataset_path=<dataset path> per_device_batch_size=1 enable_checkpointing=false enable_profiler=false remat_policy=full global_parameter_scale=4 steps=300 max_target_length=2048 use_iota_embed=true reuse_example_batch=1 dataset_type=synthetic attention=flash gcs_metrics=True run_name=$(USER)-pw-xpk-test-1'
```

Comment:
Curious if you have a simpler example. This seems like a complicated README case example to maintain.

Like in the non-pathways workload command example we just print hello world. Is that possible here too?

Reply:
Thanks for the question, Victor! Let me check if other dependencies are up-to-date. If so that I can simplify it.

Reply:
> however, we really want the workload to actually train!

Agreed this is the ultimately goal but we should go one step at a time. How about we have both a simple workload and a MaxText example?

I want users to be able to verify they have xpk + system working before trying to train.